src/System.Management.Automation/engine/lang/parserutils.cs

```diff
@@ -1521,6 +1527,81 @@ public bool MoveNext()
             return true;
         }
     }
+
+    /// <summary>
+    /// The simple enumerator class is used for the range operator '..'
+    /// in expressions like 'A'..'B' | ForEach-Object { $_ }
+    /// </summary>
+    internal class CharRangeEnumerator : IEnumerator
+    {
+        private int _lowerBound;
+
+        internal int LowerBound
+        {
+            get { return _lowerBound; }
+        }
+
+        private int _upperBound;
+
+        internal int UpperBound
+        {
+            get { return _upperBound; }
+        }
+
+        private int _current;
+
+        Object IEnumerator.Current
```

Comment:
Public properties should go before Internal properties https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1202.md